### DISCUSSION TITLE
Routing for web apps (like go_router)

### DISCUSSION THREAD

**Discussion by kageeker** [FilledStacks/stacked#810] [Q&A] [answered]
Hi everyone, I am looking for a way to support routing for the web version of our app which uses the stacked navigator.
Is there a way to implement Routing on top of the already existing navigator?

Thanks :)

**Comment by FilledStacks** [comment, marked as answer]
@kageeker we have not done any RnD for this. We have no guide on how to do this at the moment. We're currently not focused on Flutter web apps, only mobile and desktop apps.

**Reply by kageeker** [reply]
@FilledStacks Thanks for getting back to me, At least now I am know there is isn't an easy way of doing this.
I will try to do make this work & I will be happy to share my findings with the community.

**Reply by FilledStacks** [reply]
@kageeker that would be very much appreciated! You can share as much details as you want or only an example app. That would be a great starting point for us.
